### ISSUE TITLE
document_embeddings gives an unexpected error using RoBERTa model

### ISSUE THREAD

**Issue by trokhymovych** [flairNLP/flair#1366]
document_embeddings gives an error "IndexError: index 0 is out of bounds for dimension 0 with size 0" for some sentences using RoBERTa model. 

It appears using both DocumentRNNEmbeddings and DocumentPoolEmbeddings 

**To Reproduce**

```
from flair.embeddings import WordEmbeddings, FlairEmbeddings, DocumentPoolEmbeddings, Sentence, DocumentRNNEmbeddings
from flair.embeddings import RoBERTaEmbeddings
embedding = RoBERTaEmbeddings(pooling_operation="mean")
document_embeddings_roberta = DocumentRNNEmbeddings([embedding]) #, fine_tune_mode='nonlinear')
s = 'negative reconnaissance it' 
sentence = Sentence(s)
document_embeddings_roberta.embed(sentence)
vector = sentence.get_embedding()
vector
```

**Expected behavior**
I expect to get a vector of embedding in the form of Tensor. 

**Screenshots**
![image](https://user-images.githubusercontent.com/33522867/72734118-d22a9180-3ba1-11ea-8495-939103636637.png)
![image](https://user-images.githubusercontent.com/33522867/72735254-1dde3a80-3ba4-11ea-9c55-905756d5d052.png)

## However...
If I add a "." at the end of the sentence - it is starting to perform as expected.
![image](https://user-images.githubusercontent.com/33522867/72734383-38171900-3ba2-11ea-96e5-89fd9411584d.png)
code:
```
from flair.embeddings import WordEmbeddings, FlairEmbeddings, DocumentPoolEmbeddings, Sentence,DocumentRNNEmbeddings
from flair.embeddings import RoBERTaEmbeddings
embedding = RoBERTaEmbeddings(pooling_operation="mean")
document_embeddings_roberta = DocumentRNNEmbeddings([embedding]) #, fine_tune_mode='nonlinear')
s = 'negative reconnaissance it.' 
sentence = Sentence(s)
document_embeddings_roberta.embed(sentence)
vector = sentence.get_embedding()
vector
```

**Environment (please complete the following information):**
Ubuntu
![image](https://user-images.githubusercontent.com/33522867/72735009-a7413d00-3ba3-11ea-9320-c6befc904f1f.png)

**Additional context**
What is interesting that there is no such behavior using the BERT model.

**Comment by stefan-it** [comment]
Hi @trokhymovych ,

could you try use the latest `master` version of Flair? I did some tokenization fixes for the GPT2-based models :)

**Comment by trokhymovych** [comment]
Thank you, @stefan-it 
It is working with latest `master`

**Comment by alanakbik** [comment]
We've just released Flair 0.5, where you can get document embeddings directly out of the transformer: 

```python
from flair.embeddings import TransformerDocumentEmbeddings

# init embedding
embedding = TransformerDocumentEmbeddings('roberta-base')

# create a sentence
sentence = Sentence('The grass is green .')

# embed the sentence
embedding.embed(sentence)
```
